### PR TITLE
use modern API in addFavIcon

### PR DESCRIPTION
**Pull request by synox**
Based on https://javascript.info/modifying-document#summary these are also “old school” methods.

Simplifies the code, easier to understand.

### CODE REVIEW

File: scripts/scripts.js [hunk 1]
```diff
@@ -83,10 +83,9 @@ export function addFavIcon(href) {
   link.href = href;
   const existingLink = document.querySelector('head link[rel="icon"]');
   if (existingLink) {
-    existingLink.parentElement.replaceChild(link, existingLink);
+    existingLink.replaceWith(link);
   } else {
-    document.getElementsByTagName('head')[0].appendChild(link);
-  }
```
Comment (rofe) [inline scripts/scripts.js:89]: I think this bracket should have stayed

**Review by rofe — CHANGES_REQUESTED** [review]

File: scripts/scripts.js (hunk 1 above)
Comment (synox) [reply scripts/scripts.js:89]: oops! fixed

**Comment by synox** [comment]
@rofe done

**Comment by rofe** [comment]
@synox great, now all that's left to do is adding the test URL to the description for the last check to turn green.

**Comment by rofe** [comment]
Changes included in #214.

**Pull request closed without merging**